python/src/sdf/pyRoot.cc

```diff
@@ -83,7 +83,7 @@ void defineRoot(pybind11::object module)
          "Get a world based on an index.")
      .def("world_name_exists", &sdf::Root::WorldNameExists,
           "Get whether a world name exists.")
-    .def("model", &sdf::Root::Model,
+    .def("model", pybind11::overload_cast<sdf::Model *>(&sdf::Root::Model),
```

Comment:
there are build failures related to this line. In the [binding for World::PhysicsByIndex](https://github.com/gazebosim/sdformat/blob/0e227a6d0c2f2b7e6172916aaf7d97eb8e75ed9e/python/src/sdf/pyWorld.cc#L213-L215), the template argument to `pybind11::overload_cast` is `uint64_t`, which matches the input argument to that function not its return type. Maybe try reverting this line?